File: MODUL 6 PEMROGRAMAN 1/PRAK605-2310817120002-SITI RATNA DWINTA SARI.py
n = int(input())
A = []
B = []
hasil = []

print("Matriks A")
for i in range(n):
    A.append(list(map(int, input().split())))

print("Matriks B")
for i in range(n):
    B.append(list(map(int, input().split())))

print("Matriks AXB:")
for i in range(n):
    hasil.append([0] * n)
    for j in range(n):
        for k in range(n):
            hasil[i][j] += A[i][k] * B[k][j]
        print(hasil[i][j], end=" ")
    print()

File: MODUL 6 PEMROGRAMAN 1/PRAK605-2310817120002-SITIRATNADWINTASARI.c
#include <stdio.h>

int main() {
    int A[100][100], B[100][100], hasil[100][100], n;
    scanf("%d", &n);

    printf("Matriks A\n");
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            scanf("%d", &A[i][j]);
        }
    }
    printf("Matriks B\n");
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            scanf("%d", &B[i][j]);
        }
    }
    printf("Matriks AXB:\n");
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < n; j++) {
            hasil[i][j] = 0;
            for (int k = 0; k < n; k++) {
                hasil[i][j] += A[i][k] * B[k][j];
            }
            printf("%d ", hasil[i][j]);
        }
        printf("\n");
    } return 0;
}
